### ISSUE TITLE
Is the full dataset with 527 houses available 

### ISSUE THREAD

**Issue by xuyongzhi** [woodfrog/floor-sp#4]
Thank you so much for sharing the amazing work! It seems only the 100 houses used for training are opened and the other 427 for training are not. May I ask do you have a plan to open the whole dataset. Thanks a lot!

**Comment by woodfrog** [comment]
Hi, 
Thank you for your interest!  As explained in the README, the 100 public samples include the full test set used in the paper. Beike decided not to release the training set for business and privacy concerns, so we could only provide the model's checkpoints rather than the full training set. Sorry for the inconvenience. 

If you want to try new methods based on primitive detection, the number of instances for corner/edge/room in the 100 released samples should be enough for training some basic primitive detection models.

**Comment by xuyongzhi** [comment]
Thanks for your quick reply and nice suggestion!
